tests/unit/src/main/scala/tests/debug/ClientProxy.scala

```diff
@@ -0,0 +1,50 @@
+package tests.debug
+
+import org.eclipse.lsp4j.debug._
+import org.eclipse.lsp4j.debug.services.IDebugProtocolClient
+
+trait ClientProxy extends IDebugProtocolClient {
```

Comment:
Why do we need this class? We were talking about just forwarding messages without any actual knowledge of the protocol.

Reply:
Nvm, it's for tests. I am blind officially.